pootle/static/css/editor.css

```diff
@@ -1022,13 +1028,43 @@ html[dir="rtl"] .suggestion-original
 .editor-area-wrapper
 {
     background-color: #fff;
-    border: 1px solid #d9d9d9;
-    border-radius: 3px;
     box-shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.075) inset;
-    margin: 0 auto 0.5em;
     overflow: hidden;
+    border-right: 1px solid #d9d9d9;
+    border-left: 1px solid #d9d9d9;
+    border-top: 1px solid #d9d9d9;
+}
+
+.editor-area-wrapper:last-child {
+    border-bottom: 1px solid #d9d9d9;
+    border-bottom-left-radius: 3px;
+    border-bottom-right-radius: 3px;
+}
+
+.editor-area-wrapper:first-child {
+    border-top: 1px solid #d9d9d9;
+    border-top-left-radius: 3px;
+    border-top-right-radius: 3px;
+}
+
+.plural-form-label
+{
+    color: #c896c8;
+    font-size: 85%;
+    margin-top: 0.5em;
+}
+
+.editor-area-wrapper textarea::-webkit-input-placeholder
+{
+    color: #c896c8;
 }
 
+.editor-area-wrapper textarea:-moz-placeholder
```

Comment:
I find cleaner to merge this declaration with the previous one. Also why `:` is being used here instead `::`?